### DISCUSSION TITLE
How to use with Svelte?

### DISCUSSION THREAD

**Discussion by kenny101** [khmyznikov/pwa-install#51] [Q&A]
Hi, I have a PWA built using sveltekit. I import the PWAInstallElement but there appears to be a type error.  `Argument of type 'typeof PWAInstallElement' is not assignable to parameter of type 'ConstructorOfATypedSvelteComponent'.` What's the correct way to use this with svelte? Thanks

**Comment by khmyznikov** [comment]
@kenny101 Hi! Can you create a glitch project with your version of svelte, I can try to mix it and see what's the problem there?

**Reply by khmyznikov** [reply]
Maybe this one also can help you: https://css-tricks.com/using-custom-elements-in-svelte/
